packages/ses/src/evaluate.js

```diff
@@ -53,11 +55,16 @@ export function performEval(
   } finally {
     if (scopeHandler.useUnsafeEvaluator === true) {
       // The proxy switches off useUnsafeEvaluator immediately after
-      // the first access, but if that's not the case we abort.
-      throwTantrum('handler did not revoke useUnsafeEvaluator', err);
-      // If we were not able to abort, at least prevent further
-      // variable resolution via the scopeHandler.
+      // the first access, but if that's not the case we should abort.
+      // This condition is one where this vat is now hopelessly confused,
+      // and the vat as a whole should be aborted. All immediately reachable
+      // state should be abandoned. However, that is not yet possible,
+      // so we at least prevent further variable resolution via the
+      // scopeHandler, and throw an error with diagnostic info including
+      // the thrown error if any from evaluating the source code.
       scopeProxyRevocable.revoke();
+      // TODO A GOOD PLACE TO PANIC(), i.e., kill the vat incarnation.
```

Comment:
TODO

Reply:
Left that line in but added a link to https://github.com/Agoric/SES-shim/issues/490
I think that resolves this well enough for now.